public/doupload.php

```diff
@@ -23,26 +21,28 @@
     $requestType = mb_substr($_FILES["file"]["name"], 0, -4);
 }
 
-switch ($requestType) {
-    case "uploadbadgeimage":
-        $uploadResponse = UploadBadgeImage($_FILES["file"]);
-        $response['Success'] = $uploadResponse['Success'];
-        unset($uploadResponse['Success']);
-
-        if ($uploadResponse['Error']) {
```

Comment:
How is there not a simple macro for this already, like move(obj&, obj&); js is clever I bet it exists

Reply:
not sure i follow 😊 you mean to move over the values of `$uploadResponse` to `$response`?

Reply:
I guess I mean like std::move(), shorthand for 
if (isset(x))
{
 copy(y,x);
 unset (y);
}

Reply:
It's not a problem btw, just speculating